tests/models/test_dofa.py

```diff
@@ -107,6 +107,14 @@ def test_transforms(self, weights: WeightsEnum) -> None:
         }
         weights.transforms(sample)
 
+    def test_export_transforms(self, weights: WeightsEnum) -> None:
+        """Test that the transforms have no graph breaks."""
+        torch = pytest.importorskip('torch', minversion='2.6.0')
+        torch._dynamo.reset()
```

Comment:
Resets need to happen because torch exporting caches results and the cache will OOM for many consecutive of the same transform being exported in tests.

Reply:
I saw that too, couldn't tell if it was the same or not. As long as we don't run OOM in CI, we definitely won't locally.

Reply:
Yeah this has never happened for me when exporting one-off models, but exporting the DOFAHuge models consume most of the runner's memory.